jenkins-pipelines/feature-raft-double-cluster-size-12h.jenkinsfile

```diff
@@ -0,0 +1,13 @@
+#!groovy
```

Comment:
Can you please put this pipeline file under "raft" folder?

Reply:
Also, I don't think it should be called "feature-raft".
The scenario is valid also without raft.

And we should have a pipeline also without the experimental raft for the regular longevities.

Reply:
renamed